packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -500,6 +515,7 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
 
     if (widget.enableSearch) {
       currentHighlight = search(filteredEntries, _textEditingController);
+      scrollToHighlight();
```

Comment:
If currentHighlight is null should we still call scrollToHighlight()?

Reply:
No, I should add a null check here. Thanks for catching this! Just updated.